example.dae

```diff
@@ -101,6 +101,11 @@ global {
     # Multipath TCP (MPTCP) support. If is true, dae will try to use MPTCP to connect all nodes, but it will only take
     # effects when the node supports MPTCP. It can use for load balance and failover to multiple interfaces and IPs.
     mptcp: false
+
+    # The maximum bandwidth for accessing the Internet. It is useful for some specific protocols (e.g., Hysteria2), 
+    # which will perform better with bandwith information provided. The unit is **byte** per second.
+    bandwidth_max_tx: 26214400 # 200Mbps == 25MB/s == 26214400 B/s uplink
```

Comment:
> Do not easily add global parameters. Besides, there are not many protocols that support bandwidth settings (as far as I know). It is sufficient to design a bandwidth query parameter in the URI schema.

@douglarek Thanks for the review.

The upstream PR also includes supports for importing `maxTx` and `maxRx` from the URI scheme, and these have higher priority over global configurations.

There are several reasons to consider adding bandwidth settings as a global option:
1. When using subscriptions with URIs provided by external sources, you might not have control over the URI parameters they include.
2. In principle, a URI should contain only server-related details, ensuring it can be shared across devices without modifications. Bandwidth preferences, on the other hand, are client-specific and shouldn’t be universally shared.

Reply:
> Regarding what you mentioned in point 1, I think the idea is good, but its actual implementation may not be perfect. If you have two subscriptions and their bandwidths are different, this global setting would not be useful. 

@douglarek Thank you for your response. It's important to highlight that the `maxTx` and `maxRx` parameters are not standardized within the [Hysteria2 URI specification](https://v2.hysteria.network/zh/docs/developers/URI-Scheme/). Consequently, it's unlikely that most subscription providers will include these parameters, nor should they. The scenario where a transmission rate parameter would be relevant is when the subscription server is under your control. Therefore, the issue of bandwidth differences between subscriptions typically would not arise.

Reply:
> Adding a parameter globally that is currently only used by hy2 somewhat feels like cleaning up after hy2 itself. 

~~Well, technically 🤓👆~~, there are other protocols that can utilize the Brutal CC algorithms. For instance, the [tuic protocol](https://github.com/daeuniverse/outbound/tree/main/protocol/tuic/congestion) could potentially incorporate these algorithms, as we effectively share code base between the two protocols. However, this hasn’t been implemented for tuic since it has been archived and is now a less active protocol. In theory, all protocols that use QUIC as the transport layer could benefit from Brutal CC, even though we are not currently implementing it.

> Let me nitpick a bit more: why use maxTx and maxRx instead of up and down as parameter names?

I selected these parameter names somewhat arbitrarily to align with the naming convention used in the hy2 protocol implementation. However, I discovered that some other clients utilize different names for bandwidth parameters. 

For example, in the implementation found [here](https://github.com/MatsuriDayo/NekoBoxForAndroid/blob/2743fcb3f2208f2189d86eb2a9d4655000bcf8fb/app/src/main/java/io/nekohasekai/sagernet/fmt/hysteria/HysteriaFmt.kt#L37-L42): they use `upmbps` and `downmbps` for transmission parameters. (seems standard for `sing-box` family implementations)

I am open to updating the code to use `upmbps` and `downmbps` terms if you agree with the change.

Reply:
> I am open to updating the code to use upmbps and downmbps terms if you agree with the change.

The current names (maxTx,maxRx) are good, they are flexible so that we can support different units later.

> Adding a parameter globally that is currently only used by hy2 somewhat feels like cleaning up after hy2 itself.

Although it has not been implemented, protocols such as kcp also support fixed bandwidth. I have reason to believe that more protocols with it in the future.

In addition, if there is really more demand, we can add an override option for the group, just like what PR #623 does.